app/web/features/search/SearchPage.tsx

```diff
@@ -73,22 +74,29 @@ export default function SearchPage({
 
   // State
   const [wasSearchPerformed, setWasSearchPerformed] = useState(false);
-  const [locationResult, setLocationResult] = useState({
+  const [locationResult, setLocationResult] = useState<
+    GeocodeResult | undefined
+  >({
     bbox: bbox,
     isRegion: false,
-    location: { lng: undefined, lat: undefined },
+    location: new LngLat(
+      (parseFloat(bbox[0].toString()) + parseFloat(bbox[2].toString())) / 2,
+      (parseFloat(bbox[1].toString()) + parseFloat(bbox[3].toString())) / 2
+    ), // turns the coordinates into latitude and longitude - means the map will center even if only bbox is provided
```

Comment:
this is cool, what happens is that in the `mapWrapper` component it has some hardcoded coordinates which are the initial value (independently of the original state value)

this value is used by default, unless there is another value (when a user searches from the dashboard) then after the first load the bbox coordinates are loaded, so it has no effect on the values that are loaded by default. (I feel bad typing this comments but it's how the map is wired 😅)

Reply:
![image](https://github.com/user-attachments/assets/9e41861b-d93b-4fe3-821b-f6889364bbbc)

Reply:
No worries maps are complicated.